src/components/PathwayCourse/index.js

```diff
@@ -132,12 +131,16 @@ function PathwayCourse() {
   const [openSnackbar, setOpenSnackbar] = useState(false);
   const [openModal, setOpenModal] = useState(false);
   const [certificate, setCertificate] = useState("");
-  let completedAll = pathwayId==8?completedPortion?.total>=80: completedPortion?.total === 100;
+  let completedAll =
+    pathwayId == 8
+      ? completedPortion?.total >= 80
+      : completedPortion?.total === 100;
```

Comment:
Few comments here:
1. Using `==` will add lint. Maybe use `parseInt(pathwayId) === 8` instead.
2. We don't want to be hardcoding ids or minimum completed percentages here as they can change. We could remove the reference to the id here of `8` and the minimum completion percentage of `80` if we instead have an object mapping ids to the values necessary to complete them. If the id isn't present, we could use `100` (e.g., `const MINIMUM_COMPLETE_PERCENTAGES = {'9': 80}`) . This would allow us to easily change minimum completed portions of other pathways as well without touching the code. However, it would be better to get this map of completed percentages from the back-end. For now (after importing this constant):
```suggestion
    completedPortion?.total >= (MINIMUM_COMPLETE_PERCENTAGES[pathwayId] || 100);
```

Reply:
For now we are going for `pathway.code`. After it's releasing it will come from the backend